rtmp-client/src/main/cpp/librtmp/rtmp.h

```diff
@@ -102,6 +102,40 @@ extern "C"
 #define RTMP_PACKET_SIZE_SMALL    2
 #define RTMP_PACKET_SIZE_MINIMUM  3
 
+  typedef enum RTMPErrors_ {
```

Comment:
`RTMPResult` is probably more suitable for a name, since there are also non-error values in here. Then constants could be:

```
RTMP_SUCCESS
RTMP_READ_DONE
RTMP_ERROR_xxx
RTMP_ERROR_xxx
RTMP_ERROR_xxx
```

I.e. Start all of the error ones with `RTMP_ERROR_`, to make it clear those are errors.

Reply:
Done.